templates/part.eventsrepeat.php

```diff
@@ -0,0 +1,635 @@
+<option value="342">342</option>
+<option value="343">343</option>
+<option value="344">344</option>
+<option value="345">345</option>
+<option value="346">346</option>
+<option value="347">347</option>
+<option value="348">348</option>
+<option value="349">349</option>
+<option value="350">350</option>
+<option value="351">351</option>
+<option value="352">352</option>
+<option value="353">353</option>
+<option value="354">354</option>
+<option value="355">355</option>
+<option value="356">356</option>
+<option value="357">357</option>
+<option value="358">358</option>
+<option value="359">359</option>
+<option value="360">360</option>
+<option value="361">361</option>
+<option value="362">362</option>
+<option value="363">363</option>
+<option value="364">364</option>
+<option value="365">365</option>
+<option value="366">366</option>
+						</select><button type="button" class="ui-multiselect ui-widget ui-state-default ui-corner-all" title="Select days" aria-haspopup="true" style="width: 66px;"><span class="ui-icon ui-icon-triangle-2-n-s"></span><span>285</span></button>and the events day of year.					</td>
+				</tr>
+			</tbody></table>
+			<table style="width:100%">
+				<tbody><tr id="advanced_bymonthday">
+					<th width="75px"></th>
+					<td id="bymonthdaycheckbox">
+						<select id="bymonthday" name="bymonthday[]" multiple="multiple" title="Select days">
+							<option value="1">1</option>
+<option value="2">2</option>
+<option value="3">3</option>
+<option value="4">4</option>
+<option value="5">5</option>
+<option value="6">6</option>
+<option value="7">7</option>
+<option value="8">8</option>
+<option value="9">9</option>
+<option value="10">10</option>
+<option value="11">11</option>
+<option value="12">12</option>
+<option value="13" selected="selected">13</option>
+<option value="14">14</option>
+<option value="15">15</option>
+<option value="16">16</option>
+<option value="17">17</option>
+<option value="18">18</option>
+<option value="19">19</option>
+<option value="20">20</option>
+<option value="21">21</option>
+<option value="22">22</option>
+<option value="23">23</option>
+<option value="24">24</option>
+<option value="25">25</option>
+<option value="26">26</option>
+<option value="27">27</option>
+<option value="28">28</option>
+<option value="29">29</option>
+<option value="30">30</option>
+<option value="31">31</option>
+						</select><button type="button" class="ui-multiselect ui-widget ui-state-default ui-corner-all" title="Select days" aria-haspopup="true" style="width: 66px;"><span class="ui-icon ui-icon-triangle-2-n-s"></span><span>13</span></button>and the events day of month.					</td>
+				</tr>
+			</tbody></table>
+			<table style="width:100%">
+				<tbody><tr id="advanced_bymonth">
+					<th width="75px"></th>
+					<td id="bymonthcheckbox">
+						<select id="bymonth" name="bymonth[]" multiple="multiple" title="Select months">
+							<option value="January">January</option>
```

Comment:
I don't know exactly what the "multiselect issue" is, but in case it is useful here: we use a multiselect directive for angular in the tasks app for selecting different categories. Maybe you want to have a look:
https://github.com/owncloud/tasks/blob/master/templates/part.details.php#L106-L112
https://github.com/owncloud/tasks/tree/master/js/vendor/angularui/ui-select